### PR TITLE
Added some new tests

### PR DESCRIPTION
**Pull request by leonardovillela**
- Added some new tests for Video domain.
- Added test for VideoService delete method.
- Added build folder generated by intellij in .gitignore.

### CODE REVIEW

**Comment by leonardovillela** [comment]
Any problem with my PR?

**Comment by dularion** [comment]
ah nothing at all. thanks so much!

File: src/test/groovy/streama/VideoServiceSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      Video video = Mock(Video)
+      video.files >> [file1, file2]
+
+      ViewingStatus.findAllByVideo(video) >> [viewingStatus1, viewingStatus2]
+
+    when: 'Method deleteVideoAndAssociations is called'
+      service.deleteVideoAndAssociations(video)
+
+    then: 'The video has been flagged to delete'
+      1 * video.setDeleted(true)
```
Comment (dularion) [inline src/test/groovy/streama/VideoServiceSpec.groovy]: ive never seen this assert syntax. what does it do exactly?

Comment (leonardovillela) [inline src/test/groovy/streama/VideoServiceSpec.groovy]: Hello @dularion, this is assertion syntax in mock objects, in this form, i expected the method setDeleted will be called one time with param true.

If you want more information: http://spockframework.org/spock/docs/1.1-rc-3/interaction_based_testing.html

**Comment by dularion** [comment]
Thanks mate!

**Comment by leonardovillela** [comment]
Thanks you, i hope to contribute even more, it's an awesome project.

**Pull request merged**